handler.go

```diff
@@ -0,0 +1,17 @@
+package otelutil
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func WithSpanStartAttrs(attrs ...attribute.KeyValue) otelhttp.Option {
+	return otelhttp.WithSpanOptions(trace.WithAttributes(attrs...))
```

Comment:
since we are exposing them in otelutil, it is better to use them from the package

```suggestion
return otelhttp.WithSpanOptions(WithAttributes(attrs...))
```